src/LibraryViewExtensionMSWebBrowser/LibraryViewExtensionMSWebBrowser.csproj

```diff
@@ -285,11 +291,4 @@
       <LastGenOutput>Resources.en-US.Designer.cs</LastGenOutput>
     </EmbeddedResource>
   </ItemGroup>
-  <!-- Copy the library view extension manifest file to the default folder for Dynamo-->
-  <Target Name="AfterBuildOps" AfterTargets="Build">
-    <ItemGroup>
-      <ExtensionDefinition Include="Library_ViewExtensionDefinition.xml" />
-    </ItemGroup>
-    <Copy SourceFiles="@(ExtensionDefinition)" DestinationFolder="$(OutputPath)\viewExtensions\" />
-  </Target>
```

Comment:
I dont think you intend to touch this file correct? The view extension should not be under the output folder directly but in `viewExtensions` folder

Reply:
@QilongTang for this case I took as a base your PR for switching from cefSharp to WebBrowser, in this case I can see you inserted this lines in the LibraryViewExtensionMSWebBrowser.csproj file for copying the Library_ViewExtensionDefinition.xml to the viewExtensions folder, If I left this line both projects LibraryViewExtensionMSWebBrowser.csproj  and LibraryViewExtensionWebView2.csproj will be copying the same file two times, then depending which is compiled first is the one that will remain (so the wrong viewExtension for Library could be loaded), let me know if it makes sense to you.
This is the PR: https://github.com/DynamoDS/Dynamo/pull/11957/files

Reply:
OK gotcha, since we will obsolete and remove this view extension soon. This is good

Reply:
@QilongTang yes, I will create a task for removing the old Library extensions.
- LibraryViewExtension
- LibraryViewExtensionMSWebBrowser.